cypress/integration/scenarios/with_default_options.spec.ts

```diff
@@ -33,7 +34,28 @@ describe('Menu testing', () => {
     // Check Epinio's side menu
     epinio.checkEpinioNav();
   });
-});
+
+  it('Verify Welcome Screen without Namespaces', () => {
+    cy.clickEpinioMenu('Namespaces');
+    // Deletes all namespaces if detected
+    cy.get("body").then(($body) => {
+      if ($body.text().includes('Delete')) {
+        cy.deleteAllNamespaces()
+      }
+     }
+    )
+    cy.clickEpinioMenu('Applications');
+    cy.get('h1').contains('Welcome to Epinio', { timeout: 4000 }).should('be.visible')
```

Comment:
Have you tried without the timeout? because the default one is 10000?
https://github.com/epinio/epinio-end-to-end-tests/blob/main/cypress.json#L3